Formula/grumpy.rb

```diff
@@ -0,0 +1,25 @@
+class Grumpy < Formula
+  desc "Python to Go source code transcompiler and runtime"
+  homepage "https://github.com/google/grumpy"
+  head "https://github.com/google/grumpy.git"
+
+  depends_on "go" => :build
+
+  def install
+    system "make"
+    system "make", "DESTDIR=#{prefix}", "install"
+    system "mkdir", "-p", "#{bin}"
+    system "cp", "#{prefix}/usr/bin/grumpc", "#{bin}"
+    system "cp", "#{prefix}/usr/bin/grumprun", "#{bin}"
```

Comment:
These copies can be changed to `bin.install`
http://www.rubydoc.info/github/Homebrew/brew/master/Formula:bin